deploy/helm/sumologic/conf/setup/main.tf

```diff
@@ -1,6 +1,6 @@
 terraform {
   required_providers {
-    sumologic  = "= 2.3.0"
+    sumologic  = "= 2.3.3"
```

Comment:
I believe we should automatically apply versions with fixes

Reply:
good point, changed to `~> 2.3.0`